mosaic-site/src/components/app-header/AppHeader.module.css

```diff
@@ -0,0 +1,53 @@
+:global(header):has(.root) {
+  --navbar-padding-vertical: var(--salt-size-unit);
```

Comment:
could we rename this token, i only worry with this one in case if we ever have a component called navbar

Reply:
Good spot.
The way to override mosaic styles is via a Vanilla Extract theme contract..
We should have a huddle around Mosaic/Salt theming.

Added @DavieReid who can help us

This does not need to be a blocker for this PR, as we could iterate on this.
Theme convergence is one of the things which will come out of aligning the common parts of a site theme.

Reply:
Makes sense, I've renamed this for now to avoid any confusion

Reply:
@mark-tate @DavieReid  may be worth prefixing mosaic tokens with --mosaic, has that been considered?

Reply:
I have a PR up at the moment that will scope the mosaic variables (not prefixing, but the end result will be the same)